CONTRIBUTING.md

```diff
@@ -189,6 +189,14 @@ Requirements:
 - Mock external service calls
 - Test argument validation
 
+#### Cancellation plumbing
+
+To ensure the product code and unit tests can be cancelled quickly, contributors are required to write async methods (any returning `Task`, `ValueTask`, generic variants of those, etc.) to accept and invoke async methods with a `System.Threading.CancellationToken` parameter. The latter is enforced with the [CA2016 analyzer](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2016).
```

Comment:
this new section should also be added as DO/DO-NOT rules to https://github.com/microsoft/mcp/blob/main/servers/Azure.Mcp.Server/docs/new-command.md

Reply:
Resolving as fixed.